### PR TITLE
v0.16.0 

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: _posts/2021-06-04-v0.16-gleam-compiles-to-javascript.md [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+author: Louis Pilfold
+title: v0.16 - Gleam compiles to JavaScript!
+tags:
+  - Release
+---
+
+## Show me!
+
+What's the introduction of a new way to write front end web code without some
+clique examples?
```
Comment (michallepicki) [inline _posts/2021-06-04-v0.16-gleam-compiles-to-javascript.md:11]: > clique

Should that say `cliché`?

Comment (lpil) [reply _posts/2021-06-04-v0.16-gleam-compiles-to-javascript.md:11]: It should! Thanks!

File: _posts/2021-06-04-v0.16-gleam-compiles-to-javascript.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+Webpack.
+
+Currently a Gleam build tool is being worked on, one that will be suitable for
+use with Erlang and JavaScript, and handle the resolution and compilation of
+dependancies from the Hex package manager. 
+
+### Concurrency exploration
+
+In JavaScript code yield points have to be manually inserted using
+`Promise.prototype.then`, or with the `await` keyword, while in Erlang Go this
```
Comment (michallepicki) [inline _posts/2021-06-04-v0.16-gleam-compiles-to-javascript.md:174]:
```suggestion
`Promise.prototype.then`, or with the `await` keyword, while in Erlang or Go this
```

Comment (lpil) [reply _posts/2021-06-04-v0.16-gleam-compiles-to-javascript.md]: Thank you!

**Pull request merged**